tests/kagome/0001-parachains-pvf-code-size-bad.zndsl

```diff
@@ -0,0 +1,13 @@
+Description: PVF code size
+Network: ./0001-parachains-pvf-code-size-bad.toml
+Creds: config
+
+# Check authority status.
+alice: reports node_roles is 4
+
+# Ensure parachains are registered.
+alice: parachain 2000 is registered within 60 seconds
+
+# Ensure parachains made progress.
+alice: parachain 2000 block height != 1 within 60 seconds
+alice: log line matches glob "*Trap: final > memory:*" within 60 seconds
```

Comment:
We should probably add a comment that this is a temporary solution until Kagome team adds `parachain_candidate_validation_code_size`